vendor/engines/secure_rooms/app/models/secure_rooms/ability_extension.rb

```diff
@@ -11,6 +11,7 @@ def initialize(ability)
     def extend(user, resource)
       ability.can :manage, CardReader if user.manager_of?(resource) || user.facility_senior_staff_of?(resource)
       ability.can :index, Occupancy if user.operator_of?(resource)
+      ability.can :index, Occupancy
```

Comment:
This seems like it's giving more power than is necessary. Why doesn't the previous line handle this?